Software/gopigo3_power.py

```diff
@@ -6,17 +6,21 @@
 # Released under the MIT license (http://choosealicense.com/licenses/mit/).
 # For more information, see https://github.com/DexterInd/GoPiGo3/blob/master/LICENSE.md
 #
-# This code is for power management on a Raspberry Pi
-# GPIO 23 needs to remain low impedance (output) set to a HIGH state. If GPIO 23 gets left floating (high impedance) the GoPiGo3 assumes the RPi has shut down fully. SW should never write GPIO 23 to LOW or set it as an INPUT.
-
+# This code is for power management on a Raspberry Pi with GoPiGo3.
+#
+# GPIO 22 will be configured as input with pulldown. If pulled high, the RPi will halt.
+#
+# GPIO 23 needs to remain low impedance (output) set to a HIGH state. If GPIO 23 gets
+# left floating (high impedance) the GoPiGo3 assumes the RPi has shut down fully.
+# SW should never write GPIO 23 to LOW or set it as an INPUT.
 
 
 import RPi.GPIO as GPIO
 import time
 import os
 
 GPIO.setmode(GPIO.BCM)
-GPIO.setup(22, GPIO.IN)
+GPIO.setup(22, GPIO.IN, pull_up_down=GPIO.PUD_DOWN)
```

Comment:
Yes. It prevents the RPi from shutting down if gopigo3_power.py is running without a GPG3 connected.